vaccine_feed_ingest/runners/la/tableau/normalize.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import os
+import pathlib
+import re
+import sys
+from typing import List, Optional
+
+from vaccine_feed_ingest_schema import location as schema
+
+from vaccine_feed_ingest.utils.normalize import provider_id_from_name
+
+# Configure logger
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)s:%(name)s:%(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+)
+logger = logging.getLogger("la/tableau/normalize.py")
+
+
+def _get_id(site: dict) -> str:
+    maybe_provider = provider_id_from_name(site["name"])
+    if maybe_provider:
+        return f"{maybe_provider[0]}:{maybe_provider[1]}"
+
+    return site["name"]
+
+
+def _get_contacts(site: dict) -> Optional[List[schema.Contact]]:
+    contacts = []
+
+    if "phone" in site["contact"] and re.match(
+        r"^\(\d{3}\) \d{3}-\d{4}$", site["contact"]["phone"]
+    ):
+        contacts.append(
+            schema.Contact(contact_type="general", phone=site["contact"]["phone"])
+        )
+
+    if "website" in site["contact"]:
+        contacts.append(
+            schema.Contact(contact_type="general", website=site["contact"]["website"])
+        )
+
+    if len(contacts) > 0:
+        return contacts
+
+    return None
+
+
+def _get_address(site: dict) -> schema.Address:
+    return schema.Address(
+        street1=site["address"]["street1"],
+        city=site["address"]["city"],
+        state=site["address"]["state"],
+    )
+
+
+def _get_parent_organization(site: dict) -> Optional[schema.Organization]:
+    maybe_provider = provider_id_from_name(site["name"])
+    if maybe_provider:
+        return schema.Organization(id=maybe_provider[0])
+
+    return None
+
+
+def _get_links(site: dict) -> Optional[List[schema.Link]]:
+    maybe_provider = provider_id_from_name(site["name"])
+    if maybe_provider:
+        return [schema.Link(authority=maybe_provider[0], id=maybe_provider[1])]
+
+    return None
+
+
+def _get_normalized_location(site: dict, timestamp: str) -> schema.NormalizedLocation:
+    return schema.NormalizedLocation(
+        id=_get_id(site),
+        name=site["name"],
+        address=_get_address(site),
+        contact=_get_contacts(site),
+        parent_organization=_get_parent_organization(site),
+        links=_get_links(site),
+        source=schema.Source(
+            source="la:tableau",
+            id=site["name"],
+            fetched_from_uri="https://public.tableau.com/vizql/w/pharmacies_desktop/v/Pharmacies_desktop/bootstrapSession/sessions/${SESSION_ID}",
```

Comment:
You should use the user-friendly URL from the ticket [here](https://github.com/CAVaccineInventory/vaccine-feed-ingest/issues/94) vs. the one from the fetch stage, because you can't visit it without `${SESSION_ID}`